docs/configuration.rst

```diff
@@ -304,4 +304,34 @@ below:
      - True
      - Whether to enable the endpoint data collection in profiles.
 
+       .. _dd-appsec-enabled:
```

Comment:
those doc update seems unrelated, might be worth a different PR?

Reply:
Well, they're env vars we use and just found out they're not documented here, I did it in this PR because it is harmless and to avoid conflicts